.pre-commit-config.yaml

```diff
@@ -17,9 +17,7 @@ repos:
     - id: mypy
       name: mypy-fv3core
       args: [
-          --no-strict-optional,
-          --ignore-missing-imports,
-          --follow-imports, skip  # needed so we only test enabled files
+          --config-file, setup.cfg,
```

Comment:
Is this needed? https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-config-file

Reply:
I think it uses this one by default, but it's good to be explicit.